helpers/helpers.php

```diff
@@ -55,3 +55,8 @@ function toPercentage($value, $inverse = false)
 {
     return  ($inverse ? 1 - $value : $value) * 100;
 }
+
+function sanitizeJSInjection($text)
+{
+    return str_replace('/script>', '/\script>', str_replace('<script', '<\script', $text));
```

Comment:
diria que es pot fer mes facil amb el htmlentities o el htmlspecialchars

https://stackoverflow.com/questions/5986972/php-xss-sanitization

Reply:
o el strip_tags

Reply:
https://www.cloudways.com/blog/prevent-xss-exploits-using-laravel-validation-and-sanitization/

Reply:
Cert, he utilitzat el strip_tags. 👍🏽